pkgs/development/libraries/mesa/default.nix

```diff
@@ -178,14 +172,19 @@ let self = stdenv.mkDerivation {
     # Move other drivers to a separate output
     mv $out/lib/dri/* $drivers/lib/dri # */
     rmdir "$out/lib/dri"
-    mv $out/lib/lib*_mesa* $drivers/lib
+    mv -t $drivers/lib $out/lib/lib*_mesa* || true # ??
 
     # move libOSMesa to $osmesa, as it's relatively big
     mkdir -p {$osmesa,$drivers}/lib/
     mv -t $osmesa/lib/ $out/lib/libOSMesa*
 
     # now fix references in .la files
-    sed "/^libdir=/s,$out,$osmesa," -i $osmesa/lib/libOSMesa*.la
+    if ! sed "/^libdir=/s,$out,$osmesa," -i $osmesa/lib/libOSMesa*.la ; then
+      echo "FAILED" > $osmesa/failed
+    fi
+
+    mkdir -p $dev/lib
+    cp -a $out/lib/pkgconfig $dev/lib/pkgconfig
```

Comment:
I'm not sure what all is going on here and/or is necessary. I sort of hacked at this to get it to build.